internal/clients/nightfall/nightfall.go

```diff
@@ -211,6 +217,59 @@ func (n *Client) createScanRequest(items []string) nightfallAPI.ScanRequest {
 	}
 }
 
+func (n *Client) scanContent(ctx context.Context, cts []*contentToScan, requestNum int, logger logger.Logger) ([]*diffreviewer.Comment, error) {
```

Comment:
we don't have to do this for this PR (so we can get this code in :) ) but do you mind creating a ticket for adding tests for this func later? Seems pretty complex and a good place to add some unit tests when we have time.